samples/drivers/led_pwm/src/main.c

```diff
@@ -74,33 +73,25 @@ static void run_led_test(const struct device *led_pwm, uint8_t led)
 		}
 		k_sleep(FADE_DELAY);
 	}
+	LOG_INF("  Increased brightness gradually");
 	k_sleep(K_MSEC(1000));
 
 	/* Set LED blinking (on: 0.1 sec, off: 0.1 sec) */
-	LOG_INF("  Blinking on: 0.1 sec, off: 0.1 sec");
 	err = led_blink(led_pwm, led, 100, 100);
 	if (err < 0) {
 		LOG_ERR("err=%d", err);
 		return;
 	}
-	k_sleep(K_MSEC(5000));
-
-	/* Enable LED blinking (on: 1 sec, off: 1 sec) */
-	LOG_INF("  Blinking on: 1 sec, off: 1 sec");
-	err = led_blink(led_pwm, led, 1000, 1000);
-	if (err < 0) {
-		LOG_ERR("err=%d", err);
-		return;
-	}
```

Comment:
Hi @PerMac.

Please, keep this test.

Configuring the PWM for a LED is a trade-off. You want the PWM frequency as high as possible to fool eyes and camera on steady brightness levels. But on the other end, you also want the PWM configuration to support a 0.5 Hz hardware blinking which is a pretty common need. This test allows to verify that.

Concretely, on a ST board, this helps to check that the value assignated to "st,prescaler" DT property is correct.